x-pack/plugins/security_solution/common/siem_migrations/model/api/rules/rule_migration.schema.yaml

```diff
@@ -79,6 +79,70 @@ paths:
                     type: boolean
                     description: Indicates rules migrations have been updated.
 
+  /internal/siem_migrations/rules/install:
+    post:
+      summary: Installs translated migration rules
+      operationId: InstallMigrationRules
+      x-codegen-enabled: true
+      description: Installs migration rules
+      tags:
+        - SIEM Rule Migrations
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: array
+              items:
+                $ref: '../../rule_migration.schema.yaml#/components/schemas/RuleMigrationToInstall'
```

Comment:
Why do we need the UI to send all the elastic_rule information? We should be able to get everything we need from the index using only the `id`. (and filtering the `migration_id` in the path if we add it)
So an array of IDs in the body should be enough.

Reply:
done